.coveragerc

```diff
@@ -2,3 +2,8 @@
 omit = */tests/*
     cumulusci/files/*
 source=cumulusci
+[report]
+exclude_lines =
+    pragma: no cover
+    @abstract
+    def __repr__
```

Comment:
Ignoring abstract methods makes sense -- but isn't the decorator `@abstractmethod` rather than `@abstract`?

I disagree about ignoring `__repr__`. It's a common place for there to be lingering typos _because_ it's not run in the normal course of operation, and the way to avoid that is to make sure it's covered by tests. Otherwise mistakes won't be discovered until you're actually debugging, which causes friction.

Reply:
I think it is interpreted as a substring and thus supports both abstractmethods and abstractproperties.